wallet/reference/non-evm-apis/starknet-snap-api.md

```diff
@@ -0,0 +1,868 @@
+---
+description: See the Starknet Snap API reference.
+sidebar_position: 1
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Starknet Snap API
+
+When connected to the [Starknet Snap](../../how-to/use-non-evm-networks/starknet/index.md), dapps
+can use the Starknet Snap API to interact with users' Starknet accounts (for example, to send transactions).
+
+Currently, the [`get-starknet`](https://github.com/starknet-io/get-starknet) library only supports the
+following Starknet Snap API methods:
+
+- [`starkNet_extractPublicKey`](#starknet_extractpublickey)
+- [`starkNet_signMessage`](#starknet_signmessage)
+- [`starkNet_upgradeAccContract`](#starknet_upgradeacccontract)
+- [`starkNet_verifyMessage`](#starknet_verifymessage)
```

Comment:
Does this need to be updated to reflect the latest documentation changes? I.e., does `get-starknet` support a lot of these functionalities according to the new table on the Starknet index page?

Reply:
updated the docs to call this out